.vscode/scss.code-snippets

```diff
@@ -0,0 +1,404 @@
+{
+  "Block class BEM": {
+    "scope": "scss",
+    "prefix": "block-class",
+    "body": [
+      "$$block-class: \"${TM_FILENAME_BASE}\";",
+      "",
+      ".#{$$block-class} {",
+      "  // styles for $$block-class",
+      "  $1",
+      "}",
+      ""
+    ]
+  },
+  "Sytlelint disable next line": {
```

Comment:
Typo in Stylelint for this line and every other rule in this file containing the same word.